dash/testing/plugin.py

```diff
@@ -14,7 +11,7 @@
     from dash.testing.browser import Browser
     from dash.testing.composite import DashComposite, DashRComposite
 except ImportError:
-    warnings.warn("run `pip install dash[testing]` if you need dash.testing")
```

Comment:
Wouldn't we also need to put all the below definitions into the `try` block, so that without the necessary imports it won't try to define our fixtures?

That said, this still doesn't seem like it completely satisfies the request of #946 - this takes care of the "annoys with the warning" part, but not the "increases the time of test suite by unnecessary import of the module" part. As I said in the issue, it's not clear to me how one can resolve that part, but the ideal seems to me determining whether `setup.py` was called with `[testing]` or not - and only include `"pytest11": ["dash = dash.testing.plugin"]` if yes.